### PR TITLE
libsqlitejdbc: fix git hash for version 2.25.2 to fix JNI error

### PR DESCRIPTION
**Pull request by agawande**
Fixes https://github.com/named-data-mobile/android-crew-staging/pull/2#issuecomment-501521594

### CODE REVIEW

**Comment by cawka** [comment]
Not really sure how and why this solves the problem.  I incorrectly defined version of the package (should have been 3.27.2.1, not 3.25).  You just reverted to the older version and it magically works?

File: etc/shasums.txt [hunk 1]
```diff
@@ -9,5 +9,5 @@ packages/psync.rb 0.1.0_2 android e863e2802f8d2d603fbd8b1c30335729ad2b582a581705
 packages/ndnrtc.rb 0.0.2_1 android 3c8314476e186d250f93e68cbc820fc6eb78a9bef07ec43f8a9f1c5261b24b4b
 packages/openfec.rb 1.4.2_2 android aa8c46b4af5691ba8b7a0c01fff19a4915542854629914647d171495a831a6ec
 packages/webrtc.rb 59_2 android b6009a1539375412221b67ad64c584dc6e3fe9aa13ae9342067fe2e468817bba
-packages/libsqlitejdbc.rb 3.25.2.1_1 android 1a0238e1c18871fa9c8185ec0b47fc1e6b1177f4b580d0f0b5558508c6542cc9
+packages/libsqlitejdbc.rb 3.25.2.1_1 android f43d619825cee79e5420e09765b3df60f816c2c09a86743b8bb989a97c0b10e0
```
Comment (cawka) [inline etc/shasums.txt:12]: don't change this file, as it only makes sense if I upload the binary

**Comment by cawka** [comment]
I tried to make a test app and it is crashing without and with this patch.  Can you share your test app?

This is the error I'm getting (may be I'm missing something?):
```06-17 16:38:43.571  7875  7875 F DEBUG   : Abort message: 'java_vm_ext.cc:542] JNI DETECTED ERROR IN APPLICATION: JNI NewGlobalRef called with pending exception java.lang.ClassNotFoundException: Didn't find class "org.sqlite.core.NativeDB" on path: DexPathList[[zip file "/data/app/net.named_data.ice_ar.test1-EpxQQ5CA32lvEU4yieI4aQ==/base.apk"],nativeLibraryDirectories=[/data/app/net.named_data.ice_ar.test1-EpxQQ5CA32lvEU4yieI4aQ==/lib/x86, /data/app/net.named_data.ice_ar.test1-EpxQQ5CA32lvEU4yieI4aQ==/base.apk!/lib/x86, /system/lib]]'```

**Comment by agawande** [comment]
I think it works with 3.25.2 because they don't have the Window function lookup added to the Native class: https://github.com/xerial/sqlite-jdbc/blob/master/src/main/java/org/sqlite/core/NativeDB.c
but not the Java class:
https://github.com/xerial/sqlite-jdbc/blob/master/src/main/java/org/sqlite/core/NativeDB.java
leading to the error I saw. Maybe need to make a change upstream to add the Window function first?

I have not seen that error yet. 

I am testing on ndn-photo-app with the change to copy libsqlitejdbc over to jniLibs folder:
https://github.com/named-data-mobile/ndn-photo-app/pull/156

**Pull request closed without merging**